triage/models/estimators.py

```diff
@@ -0,0 +1,79 @@
+# coding: utf-8
+
+from sklearn.base import BaseEstimator, ClassifierMixin
+from sklearn.pipeline import Pipeline
+from sklearn.preprocessing import MinMaxScaler
+from sklearn.linear_model import LogisticRegression
+
+from triage.models.transformers import DsappCutOff
```

Comment:
I'm not sure about using the name 'models' for this submodule. I think we use it for too many things already. Given the inheritance structure I'm seeing in sklearn, would 'estimators' be a better name for the submodule, and 'classifiers' a better name for this file?